src/Pages/category.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const Category = () => (
-  <>
-    <h1>Category</h1>
-    <button type="button"> check status</button>
-  </>
-);
+const Category = () => {
+  const res = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
+  return (
+    <div>
+      <span>{res}</span>
+      <button type="button" onClick={() => dispatch(checkStatus())}>Check Status</button>
```

Comment:
Optional.
It would be better if you would hide the button once the message is displayed.